src/core/Normalization.helper.js

```diff
@@ -29,7 +29,26 @@ export default class NormalizationHelper {
     }
     normalized = Number(normalized.toFixed(0));
     return normalized;
+  }
 
+  /**
+   * @param {Number|null} value
+   * @returns {Number|null}
+   */
+  static normalizeNonZeroPositiveInteger (value) {
+    if (typeof value === 'string' || value instanceof String) {
+      /** If is string */
+      if (/^((\d*\.\d+)|(\d+))$/g.test(value)) {
```

Comment:
@robertdumitrescu MAINTENANCE regex is hard to debug; could we add a code documentation what we're expecting here and why?

Reply:
OK - fixed

Reply:
Still waiting for @MalcolmSmithUK input on this

Reply:
A decimal point (.) is fine.  Any #',' would be under the bracket of localisation rather than statistical unit.